ansible/roles/cri/handlers/main.yml

```
@@ -0,0 +1,12 @@
+---
+- name: restart docker
+  systemd:
+    name: docker
+    daemon_reload: yes
+    state: restarted
+
+- name: restart containered
```

This should be `containerd`

```suggestion
- name: restart containerd
```